frappe/www/login.html

```diff
@@ -17,9 +17,16 @@
 
 			<input type="text" id="login_email"
 				class="form-control" placeholder="{{
-					_('Email address or Mobile number')
-					if frappe.utils.cint(frappe.db.get_value('System Settings', 'System Settings', 'allow_login_using_mobile_number'))
-					else _('Email address') }}"
+					_('Email address or Mobile number or Username')
```

Comment:
@shridarpatil thanks for contributing!

this seems very hard to maintain / debug.

How about adding the placeholder in the `get_context` method in `login.py` so that the code looks much cleaner. Let me know if its not clear